yarn.lock

```diff
@@ -20488,7 +20487,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"victory@npm:^36.6.12, victory@npm:^36.8.2":
+"victory@npm:^36.8.2":
```

Comment:
What is this ? Try to recover it and recup the main one if you don't use it

Reply:
Yep so you can get the yarn.lock from main 👍

Reply:
Okay.

Reply:
if it's unused it's okay to remove, please do check that the other places where victory is used still work without the "victory" package though (nft details page for example)